gradle.properties

```diff
@@ -4,18 +4,18 @@ org.gradle.jvmargs=-Xmx3G
 
 projecte_version=1.0.4
 minecraft_version=1.16.3
-mappings_version=20200723-1.16.1
+mappings_version=20200916-1.16.2
 loader_version=34
-forge_version=34.0.1
+forge_version=34.1.0
 # This determines the minimum version of forge required to use ProjectE
 # Only bump it whenever we need access to a feature in forge that is not available in earlier versions
-min_forge_version=34.0.0
+min_forge_version=34.1.0
```

Comment:
There is no need to bump the min forge version unless we need something from it.